web/docs/integrations/github.md

```diff
@@ -47,12 +47,18 @@ psl=}
 
 For more info on the specific fields, check out this [Auth](../language/features#social-login-providers-oauth-20) section of the docs.
 
+If you're adding a new entity to your `.wasp` file for the first time, make sure to create the table for it in your database:
```

Comment:
"new entity ... for the first time" -> maybe I didn't get it exactly right, but whenever you add a new entity you have to run the migration, right? so maybe just "If you added a new entity to your wasp file, make sure you migrate your database schema"

"create a table" is also ok, i was just trying to stick with "migration" notation